modules/registry-refactoring/src/main/resources/appcatalog-mysql.sql

```diff
@@ -568,6 +568,85 @@ CREATE TABLE USER_COMPUTE_RESOURCE_PREFERENCE (
   PRIMARY KEY (RESOURCE_ID,USER_ID,GATEWAY_ID)
 ) ENGINE=InnoDB DEFAULT CHARSET=latin1;
 
+CREATE TABLE GROUP_RESOURCE_PROFILE (
```

Comment:
Let's make the PK GROUP_RESOURCE_PROFILE_ID, and then a unique constraint on (GATEWAY_ID, GROUP_RESOURCE_PROFILE_NAME).